community/io/src/test/java/org/neo4j/graphdb/mockfs/EphemeralFileSystemAbstraction.java

```diff
@@ -1273,16 +1224,14 @@ private void verifySize( int totalAmount )
 
             // Double size each time, but after 1M only increase by 1M at a time, until required amount is reached.
             int newSize = buf.capacity();
-            int sizeIndex = sizeIndexFor( newSize );
-            while ( capacity( sizeIndex ) < totalAmount )
```

Comment:
Do you think about any specific scenario for when continue doubling in size would be a bad idea?
I think it is a reasonable trade off for simple code.